CHANGELOG.md

```diff
@@ -48,6 +48,12 @@ _None._
 
 _None._
 
+## 11.1.0
+
+### New Features
+
+- `StatsTimeIntervalData` now accepts an optional `unit: StatsPeriodUnit` parameter that allows to describe the granularity of data fetched for a given period. [#712]
```

Comment:
This changelog should go to the "Unreleased" section. Changing "Unreleased" to a released version is part of the release process.

https://github.com/wordpress-mobile/WordPressKit-iOS/blob/24b055e8066ddfe69a4557c8db7529dbf0537ae0/CHANGELOG.md?plain=1#L7-L11